ci_framework/roles/set_openstack_containers/tasks/main.yml

```diff
@@ -42,8 +42,9 @@
   environment:
     KUBECONFIG: "{{ cifmw_openshift_kubeconfig }}"
     PATH: "{{ cifmw_path }}"
-  ansible.builtin.shell:
-    cmd: oc get pods -n {{ cifmw_install_yamls_defaults['OPERATOR_NAMESPACE'] }} -o name
+  ci_script:
+    output_dir: "{{ cifmw_set_openstack_containers_basedir }}/artifacts"
+    script: oc get pods -n {{ cifmw_install_yamls_defaults['OPERATOR_NAMESPACE'] }} -o name
```

Comment:
I think we need to quotes the strings when there are parameters, like you did on L68 in that same file for instance.